draft-ietf-mls-architecture.md

```diff
@@ -492,10 +526,9 @@ clients. In addition, there may be applications of MLS in which the group
 membership list is stored on some server associated with the Delivery
 Service.
 
-While this knowledge is not a break of authentication or
-confidentiality, it is a serious issue for privacy. In the case where
-metadata has to be persisted for functionality, it SHOULD be stored
-encrypted at rest.
+While this knowledge is not a breach of the protocol's authentication or
+confidentiality guarantees, it is a serious issue for privacy.  Applications
+should consider anonymous systems for server fanout such as Loopix {{Loopix}}.
```

Comment:
I thought this paragraph was w.r.t. the membership list, so encryption at rest makes sense. Maybe include the link to Loopix in the sentence that mentions traffic analysis?

Reply:
Will re-introduce